.github/workflows/self-hosted.yml

```diff
@@ -51,8 +51,8 @@ jobs:
       - name: Install ginkgo
         run: make install-tools
 
-      - name: Execute `make build`
-        run: make build
+      - name: Execute `make build-cnf-tests`
```

Comment:
Give a better name.
```suggestion
      - name: Build CNF tests
```

Reply:
I kind of like the explicit naming where you know exactly what's going on in the log.

Reply:
Rule number one:
 https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/
 
Bad example:
```
// Increase the counter.
counter++;
```

Reply:
Okay I'll change.  😄